BUILD.md

```diff
@@ -0,0 +1,203 @@
+
+<a id="dep-config"></a>
+## Dependency Configuration <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+
+<a id="env-deps"></a>
+### Using Environment Installed Dependencies *(Recommended)* <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+When utilizing a package manager, such as `Anaconda`, to install dependencies (see [INSTALL.md](INSTALL.md)) you will need to provide the path to the location of your installed pacakges. This is achieved by adding the path to your package install location to `Makefile.paths` (Example Below). It is important to note that in most cases you will only provide a single path for your environment. However, if you have manually installed dependencies (such as ZeroMQ or HDF5), you will need to provide each install location.
+
+```make
+# Makefile.paths
+
+# Custom Anaconda environment for Arkouda
+$(eval $(call add-path,/home/user/anaconda3/envs/arkouda))
+#                      ^ Note: No space after comma.
+```
+
+It is important to note that the path may vary based on the installation location of Anaconda (or pip if not using Anaconda) and your environment name. Here are some tips to locate the path.
+
+```commandline
+# when installing via pip
+%pip show hdf5 | grep Location
+Location: /opt/homebrew/Caskroom/miniforge/base/envs/ak-base/lib/python3.10/site-packages
+
+# when using conda - the first line of return gives the location
+%conda list hdf5
+# packages in environment at /opt/homebrew/Caskroom/miniforge/base/envs/ak-base:
+#
+# Name                    Version                   Build  Channel
+hdf5                      1.12.1          nompi_hf9525e8_104    conda-forge
+```
+
+The `chpl` compiler will be executed with `-I`, `-L` and an `-rpath` to each path.
+
+The minimum cmake version is 3.11.0, which is not supported in older RHEL versions such as CentOS 7; in these cases, cmake must be downloaded, installed, and linked as follows. Note: while any version of cmake >= 3.11.0 should work, we tested exclusively with 3.11.0:
+
+```
+# Export version number of cmake binary to be installed
+export CM_VERSION=3.11.0
+
+# Download cmake
+wget https://github.com/Kitware/CMake/releases/download/v$CM_VERSION/cmake-$CM_VERSION-Linux-x86_64.sh
+
+# Install cmake
+sh /opt/cmake-$CM_VERSION-Linux-x86_64.sh --skip-license --include-subdir
+
+# Link cmake version
+
+export PATH=./cmake-$CM_VERSION-Linux-x86_64/bin:$PATH
+```
+
+`cmake` can also be installed using conda or pip
+```commandline
+conda install cmake>=3.11.0
+
+pip install cmake>=3.11.0
+```
+
+<a id="install-deps"></a>
+### Installing Dependencies<sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+*Please Note: This step is to only be performed if you are NOT using dependencies from your env. If you attempt to use both there is a potential that version mismatches with cause build failures*. 
```

Comment:
i think `If you attempt to use both it is possible that version mismatches will cause build failures` makes more sense here

Reply:
Updated